client/src/containers/about/about.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import "./about.css";
+import TeamMember from "../../components/TeamMember/TeamMember";
+
+const About = () => {
+  return (
+    <div className="about-container">
+      <h1 className="center">About</h1>
+      <h3>Our goal</h3>
+      <div className="cardDetails">
+        <div className="subContainer">
```

Comment:
Are you using camel case or kebab case? This needs to be consistent. E.g.   <div className="about-container"> and  <div className="cardDetails"> are different.